src/apps/loyalty/server/current_user.ts

```diff
@@ -0,0 +1,13 @@
+import * as Backbone from "backbone"
+
+export default class CurrentUser extends Backbone.Model {
+  get url() {
+    return `${process.env.ARTSY_URL}/api/v1/me`
+  }
+
+  sync(method, model, options: any = {}) {
+    options.headers = options.headers || {}
+    options.headers["X-Access-Token"] = this.get("accessToken")
+    return super.sync(method, model, options)
+  }
+}
```

Comment:
[artsy-passport](https://github.com/artsy/artsy-passport) requires a CurrentUser backbone model to work properly

The keys are cased so it's convenient to pass in a configuration hash. A minimal setup could look like this:

> ```javascript
> app.use artsyPassport _.extend config,
>   CurrentUser: CurrentUser
> ```
> Note: CurrentUser must be a Backbone model with typical get and toJSON methods.